src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/_help.py

```diff
@@ -580,24 +575,28 @@
 
 helps['servicebus namespace network-rule add'] = """
     type: command
-    short-summary: Set a Network rule for a Namespace
+    short-summary: Add a network rule for a namespace.
     examples:
-        - name: set a Network rule for a namespace
-          text: az servicebus namespace network-rule add --resource-group myresourcegroup --namespace-name mynamespace
+        - name: add a VirtualNetwork rule in NetworkruleSet for a namespace
+          text: az servicebus namespace network-rule add --resource-group myresourcegroup --namespace-name mynamespace --subnet subnetId --ignore-missing-endpoint True
+        - name: add a IP rule in NetworkruleSet for a namespace
+          text: az servicebus namespace network-rule add --resource-group myresourcegroup --namespace-name mynamespace --ip-address ipmask --action Allow
 """
 
 helps['servicebus namespace network-rule list'] = """
     type: command
-    short-summary: Show properties of Network rule of the given Namespace
+    short-summary: Show properties of Network rule of the given Namespace.
     examples:
         - name: Show properties of Network rule of the given Namespace
           text: az servicebus namespace network-rule list --resource-group myresourcegroup --namespace-name mynamespace
 """
 
 helps['servicebus namespace network-rule remove'] = """
     type: command
-    short-summary: Remove Network Rule of the given Namespace
-    examples:
-        - name: Remove Network rulet of the given Namespace
-          text: az servicebus namespace network-rule remove --resource-group myresourcegroup --namespace-name mynamespace
+    short-summary: Remove network rule for a namespace
+    examples:        
+        - name: remove VirtualNetwork rule from NetworkruleSet for a namespace
+          text: az servicebus namespace network-rule remove --resource-group myresourcegroup --namespace-name mynamespace --subnet subnetId
```

Comment:
subnetId should be `{subnetId}` to clearly denote that this is a placeholder and not an actual example of a subnet ID.